### PR TITLE
Clean up overall and add pars descriptions to each module

### PR DESCRIPTION
**Pull request by deeikele**

### CODE REVIEW

File: main.bicep [hunk 1]
```diff
@@ -1,182 +1,144 @@
...
-@description('Specifies the address prefix of the azure bastion subnet.')
+@description('Bastion subnet address prefix')
 param azureBastionSubnetPrefix string = '192.168.250.0/27'
 
-// DSVM Jumpbox username and password
+@description('Jumbox virtual machine username')
 param dsvmJumpboxUsername string
+
 @secure()
+@description('Jumbox virtual machine password')
```
Comment (ezwiefel) [inline main.bicep:32]: Minor Typo

File: modules/containerregistry.bicep [hunk 2]
```diff
@@ -1,25 +1,33 @@
 // This template is used to create a Container Registry with private endpoint.
```
Comment (ezwiefel) [inline modules/containerregistry.bicep:1]: I liked the comments you put in the other modules - it looks much cleaner - you could add that here as well.

`// Creates a Container Registry with Private Link Endpoint`

File: modules/applicationinsights.bicep [hunk 3]
```diff
@@ -1,14 +1,15 @@
 // This template is used to create Application Insights.
```
Comment (ezwiefel) [inline modules/applicationinsights.bicep:1]: I liked the comments you put in the other modules - it looks much cleaner - you could add that here as well.

`// Creates Application Insights Workspace` 
(I don't know the right word for App Insights - workspace, area, resource??)

**Review by ezwiefel — CHANGES_REQUESTED** [review]
Overall, it looks much cleaner... just a few small tweaks and typo corrections requested.

File: modules/applicationinsights.bicep (hunk 3 above)
Comment (deeikele) [reply modules/applicationinsights.bicep:1]: Yeah.. instance possibly?

**Pull request closed without merging**